containers/RoundSection/index.js

```diff
@@ -67,13 +25,80 @@ const RoundSection = ({
   btnStyle,
   sectionSubTitle,
   cardArea,
-  btnTitle,
+  btnTitle
 }) => {
+  const [hasRound, setHasRound] = useState(true);
+  const [endTime, setEndTime] = useState(0);
+  const [isLoading, setLoading] = useState(true);
+
+  async function fetchData() {
+    const res = await fetch("https://api.acquity.io/v1/round/active");
+    if (res!==null){
```

Comment:
Don't need `!==` for nullness, we can directly use `if (res)` since `null` is `False` in Javascript